src/collections/_documentation/platforms/javascript/index.md

```diff
@@ -375,6 +380,17 @@ Sentry supports additional context with events. Often this context is shared amo
 
 : Arbitrary unstructured data which the Sentry SDK stores with an event sample
 
+### Adding Context to Events
+There are two ways you can add context to events.
+
+#### Use-Case: local/event-specific
+
+[ code snippet here ]
+
+#### Use-Case: global/middleware-type context
+
+[ code snippet here ]
+
```

Comment:
lets just bundle them all together

you are either:

- sending a known context (heres how)
- or sending your own custom context

we can remove "extra" context from the documentation or make it a tiny note as "setContext" is being preferred going forward

Reply:
I'm okay with bundling them together as long as how it works globally is the same as how it works locally. Is that right? They work the same?

If so, will someone (@HazAT ? @untitaker ?) provide a good code snippet to go with this section?

Reply:
They work slightly different, but to a user its "I want to add data to an event" and you can either "add context for all events that Sentry captures" or "add context to a single event" kind of thing. Will defer to other folks on code examples.